tools/docker/Dockerfile

```diff
@@ -183,9 +187,16 @@ COPY _submodules/runc/runc /usr/local/bin
 COPY --from=firecracker-vm-root-builder /output/vm.ext4 /var/lib/firecracker-containerd/runtime/default-rootfs.img
 COPY tools/docker/firecracker-runtime.json /etc/containerd/firecracker-runtime.json
 
+# pull the images the tests need into the content store so we don't need internet
+# access during the tests themselves
 COPY tools/docker/naive-snapshotter/config.toml /etc/containerd/config.toml
-COPY tools/docker/naive-snapshotter/entrypoint.sh /entrypoint
+RUN containerd 2>/dev/null & \
```

Comment:
I hate it but I don't have any suggestions that don't involve a lot more work than we want to do right now...